### DISCUSSION TITLE
New "Enable third-party controllers" option in Lightroom CC

### DISCUSSION THREAD

**Discussion by electerious** [rsjaffe/MIDI2LR#807] [General]
There's a new option "Enable third-party controllers" in the settings of Adobe Lightroom CC, which sounds like controller support is finally a part of Lightroom CC, too. It's hard to find any details, but I hope that the API will be similar to the one of Lightroom Classic, which could allow MIDI2LR support for Lightroom CSS.

![image](https://github.com/user-attachments/assets/f23d9977-350d-43d3-83f2-4a6600395d4c)

The only blog I found that mentions this feature is this one: https://www.lightroomqueen.com/whats-new-in-lightroom-2024-12/

**Comment by Seb-Kaiser** [comment]
Hi, I am also very interested in this new feature, because I'm part of the people who prefer the cloud version, even though I wished for so long it could work with a controller.

Any plans to make midi2lr compatible with Lightroom cloud, now that it is possible? 

Thank you so much in advance!

**Comment by mvcoile** [comment]
_As far as I understand it*:_ This is the same API as Lightroom CC. **However**, MIDI2LR (and LrControl) use a custom plugin/module to directly control Lightroom instead and are ignoring the API. So MIDI2LR is implicitly not compatible.

**Comment by electerious** [comment]
Made my own controller for Lightroom. It's not using MIDI, but it still might be a solution for some: https://github.com/electerious/lightroom-controller
